bot/exts/moderation/clean.py

```diff
@@ -179,11 +179,11 @@ def predicate_regex(message: Message) -> bool:
 
         def predicate_range(message: Message) -> bool:
             """Check if the message age is between the two limits."""
-            return first_limit <= message.created_at <= second_limit
+            return first_limit < message.created_at < second_limit
 
         def predicate_after(message: Message) -> bool:
             """Check if the message is older than the first limit."""
```

Comment:
Shouldn't this be "younger" rather than "older"?

Reply:
```suggestion
            """Check if the message is younger than the first limit."""
```

If it should, here's a suggestion for easy peasy one-click fix.